packages/tpetra/core/test/Map/CMakeLists.txt

```diff
@@ -189,3 +189,14 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
   NUM_MPI_PROCS 2-10
   STANDARD_PASS_OUTPUT
   )
+
+
+TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  Map_getMinGlobalIndex
+  SOURCES
+    Map_getMinGlobalIndex
+    ${TEUCHOS_STD_UNIT_TEST_MAIN}
+  COMM mpi
+  NUM_MPI_PROCS 4
```

Comment:
It would be good to test both the 1 and > 1 number of processes cases.  Try this:
```
NUM_MPI_PROCS 1-4
```
Not sure if the following works but it would avoid a couple unnecessary executable runs:
```
NUM_MPI_PROCS 1 4
```

Reply:
I made the change and I tested going from 1 to 4 mpi ranks. It works fine now.